libdispatch/dauth.c

```diff
@@ -91,8 +91,9 @@ NC_authsetup(NCauth* auth, NCURI* uri)
     char* uri_hostport = NULL;
 
     if(uri != NULL)
-	uri_hostport = NC_combinehostport(uri);    
-
+      uri_hostport = NC_combinehostport(uri);
```

Comment:
@WardF I am using indents of 3 everywhere, but this seems to use indents of 2.

I don't really care which we use, but let's all use the same indents. I have the following in my .emacs:

```
 (setq c-default-style "linux"
           c-basic-offset 3)

```

If you want to use something else, let me know so we can all use the same settings.

Reply:
I always use 4 spaces or one tab instead of 8 spaces. I personally find 3 spaces
irritating and have  occasionaly changed 3 to 4 when I change a file a lot.

Reply:
Also, since it screws up git diff when we change the spacing, I would prefer
that we do not change the spacing on any file and try to adhere to whatever
style (including braces) we find in a file.

Reply:
I just discovered the "Diff Settings" menu to near the top. That at least can
get rid of extraneous space changes.